### DISCUSSION TITLE
如何通过http等方式连通streamPark，根据微服务传递的参数，自动创建启动Flink Job

### DISCUSSION THREAD

**Discussion by phoeph** [apache/streampark#2451] [Q&A]
我们团队有一些自动创建Flink Job的需求，目前调研看到StreamPark，但是看到只能在页面上配置并启动Flink Job。
不知是否有方式可以通过http传递参数，等方式，让我们的微服务可以传递参数配置到StreamPark，自动创建和启动Flink Job。

谢谢。

**Comment by zhongqishang** [comment]
+1 for this.

**Comment by Al-assad** [comment]
Hi, one possible solution to meet this functionality is through the use of a unified REST API. There have been proposals in the community for such an API, but it has not been effectively pushed forward yet... If you are interested, we can work together to push the development of this feature.

**Reply by phoeph** [reply]
Hi @AI-assad,

Thanks for the reply.
Spring and Rest things are not my strong point, I wonder if we can use F12 to extract the http request instead? :)
We can simulate to call these http requests that we clicked the run button on StreamX's web page.
This maybe a workaround solution?
